code/__DEFINES/~skyrat_defines/DNA.dm

```diff
@@ -58,7 +58,11 @@
 #define PREVIEW_PREF_LOADOUT "Loadout"
 #define PREVIEW_PREF_NAKED "Naked"
 
-#define MANDATORY_FEATURE_LIST list("mcolor" = "FFB","mcolor2" = "FFB","mcolor3" = "FFB","ethcolor" = "FCC","skin_color" = "FED","flavor_text" = "","breasts_size" = 1,"breasts_lactation" = FALSE,"penis_size" = 13,"penis_girth" = 9,"penis_taur_mode" = TRUE,"balls_size" = 1)
+#define BODY_SIZE_NORMAL 		1.00
+#define BODY_SIZE_MAX			1.25
+#define BODY_SIZE_MIN			0.85
+
+#define MANDATORY_FEATURE_LIST list("mcolor" = "FFB","mcolor2" = "FFB","mcolor3" = "FFB","ethcolor" = "FCC","skin_color" = "FED","flavor_text" = "","breasts_size" = 1,"breasts_lactation" = FALSE,"penis_size" = 13,"penis_girth" = 9,"penis_taur_mode" = TRUE,"balls_size" = 1, "body_size" = BODY_SIZE_NORMAL)
```

Comment:
The purpose of this define?

Reply:
All features are validated through this define, and this define is used as a basis for all DNA, and DNA expects someone  to have a "body_size" feature. Basically, if a feature exists in this list, EVERYTHING will have it